src/LogProviders/IBuildSummary.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.VisualStudio.Shell.TableManager;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Tools.Providers
+{
+    public interface IBuildSummary : IComparable<IBuildSummary>
+    {
+        public int BuildId { get; }
+
+        public BuildType BuildType { get; }
+
+        public IEnumerable<string> Dimensions { get; }
+
+        public IEnumerable<string> Targets { get; }
+
+        public DateTime StartTime { get; }
+
+        public TimeSpan Elapsed { get; }
+
+        public BuildStatus Status { get; }
+
+        public string ProjectPath { get; }
```

Comment:
The front end doesn't need the full path. Consider just sending the project name, which the UI then splits into project name and project type (or splitting these out on the server side).